src/main/java/com/avispl/dal/communicator/cisco/CiscoCommunicator.java

```diff
@@ -2240,7 +2394,7 @@ private void enrichContentChannelStatsData(ContentChannelStats contentChannelSta
                 return;
             }
 
-            contentChannelStats.setCodec(video.getProtocol());
+            contentChannelStats.setCodec(video.getCodec());
 
             switch (channel.getDirection()) {
```

Comment:
are we guaranteed `channel` is not null?

Reply:
adding a check at the start of this method